lib/hash_recursive_merge.rb

```diff
@@ -1,73 +0,0 @@
-# 
-# = Hash Recursive Merge
-# 
-# Merges a Ruby Hash recursively, Also known as deep merge.
-# Recursive version of Hash#merge and Hash#merge!.
-# 
-# Category::    Ruby
-# Package::     Hash
-# Author::      Simone Carletti <weppos@weppos.net>
-# Copyright::   2007-2008 The Authors
-# License::     MIT License
-# Link::        http://www.simonecarletti.com/
-# Source::      http://gist.github.com/gists/6391/

```

Comment:
i've intentionally left out this header info (the gist link 404s now anyway), lmk if you'd prefer to include it